myst_nbs/gaussian_processes/GP-Kron.myst.md

```diff
@@ -241,11 +275,33 @@ axs = axs.ravel()
 for i, ax in enumerate(axs):
     ax.axis("off")
     ax.scatter(X[:, 0], X[:, 1], s=20, c=y, marker="o", norm=norm, cmap=cmap)
-    ax.scatter(Xnew[:, 0], Xnew[:, 1], s=20, c=ppc["fnew"][i], marker="s", norm=norm, cmap=cmap)
+    ax.scatter(
+        Xnew[:, 0],
+        Xnew[:, 1],
+        s=20,
+        c=ppc.posterior_predictive["fnew"].sel(chain=0)[i],
+        marker="s",
+        norm=norm,
+        cmap=cmap,
+    )
     ax.set_title(f"Sample {i+1}", fontsize=24)
 ```
 
+## Authors
+* Authored by [Bill Engels](https://github.com/bwengals), 2018
+* Updated by [Chris Fonnesbeck](https://github.com/fonnesbeck), [Raul-ing Average](https://github.com/CloudChaoszero), [Christopher Krapu](https://github.com/ckrapu), [Hector Munoz](https://github.com/hectormz), and [Danh Phan](https://github.com/danhphan), 2020-2022
```

Comment:
From a quick look at the [history](https://github.com/pymc-devs/pymc-examples/commits/main/examples/gaussian_processes/GP-Kron.ipynb) of the notebook after it was moved to pymc-examples, according to what we agreed on with https://docs.pymc.io/en/latest/contributing/jupyter_style.html#authorship-and-attribution and https://docs.pymc.io/en/latest/contributing/jupyter_style.html#first-cell (last sentence in the section, relevant for adding people to author section in post directive). This should be something like

```
## Authors
* Authored by [Bill Engels](https://github.com/bwengals), 2018
* Updated by [Raul-ing Average](https://github.com/CloudChaoszero), March 2021
* Updated by [Christopher Krapu](https://github.com/ckrapu), July 2021
* Updated to PyMC 4.x by [Danh Phan](https://github.com/danhphan), November 2022
```

Chris moved the notebook, which we don't list here, I added the myst counterparts and in the sync process some trailing whitespaces and things like that were modified, and Hector ran codespell which fixed a couple typos.